base/cvd/cuttlefish/host/commands/run_cvd/launch/streamer.cpp

```diff
@@ -268,7 +268,7 @@ class WebRtcServer : public virtual CommandSource,
 
   // SetupFeature
   bool Enabled() const override {
-    return sockets_.Enabled() && instance_.enable_webrtc();
+    return sockets_.Enabled() && config_.vm_manager() != VmmMode::kGem5;
```

Comment:
nit: This should be `&& (vmm == crosvm || vmm == qemu)` in case we add a new arch in the future.

Reply:
Using a `switch` would probably be better as that would case a build failure if a new arch is added to the enum but not handled here.

Reply:
Done.